lessons/pydata/api/index.ipynb

```diff
@@ -0,0 +1,1713 @@
+{
```

Comment:
<p>Já bych klidně <code>curl</code> vynechal. Času bude málo, informací hodně. Zaměřil bych to na Python ecosystem: requests, pandas.read* přímo z http(s) atp. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PyDataCZ/naucse.python.cz/pull/32/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='39'/>

Reply:
curl je tam spíš pro ukázku, že něco takového existuje a vyzkoušel bych nějaké volání jednoduché

Reply:
<p>ok</p><p> </p>